frontend/cypress/e2e/change-complaint-status.cy.ts

```diff
@@ -11,7 +11,8 @@ describe("Complaint Assign and Status Popover spec", { scrollBehavior: false },
   });
 
   Cypress._.times(complaintTypes.length, (index) => {
-    it("Changes status of open complaint to closed and back to open", () => {
+    //Skip this test as users are not allowed to close complaints from the list view for now
+    it.skip("Changes status of open complaint to closed and back to open", () => {
```

Comment:
Doesn't this mean this test doesn't do anything now?

Reply:
yeah 😅 should we remove it completely?

Reply:
I don't like this test because It grabs a random complaint and changes the status.  Why don't you update it to re-open complaint 24-000076?

Reply:
@afwilcox oh we hide the "Update status" button in the list view, so there's no button available for the test rn